### PR TITLE
Explain our branching strategy

### PR DESCRIPTION
**Pull request by sdodson**
Explain out branching strategy

### CODE REVIEW

**Comment by sdodson** [comment]
@elyscape does this help?

File: README.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ that it installs currently require a package that provides `docker`. Currently
...
 they may in the future.
 
+##Branches
+The master branch tracks our current work and should be compatible with both
+Origin master branch and the current Origin stable release, currently that's
+v1.3 and v1.2. Enterprise branches exist where we coordinate with internal Red
+Hat Quality Assurance teams. Fixes and backwards compatible feature improvements
+are often backported to the more current enterprise branches.  The enterprise
+branches should also be compatible with the corresponding Origin release for
+users who are looking for more conservative rate of change.
```
Comment (detiber) [inline README.md]: We should probably delete off some of the old branches that are not version related as well.

**Comment by elyscape** [comment]
Looks good to me, though it might also be worthwhile to explicitly call out the releases as well.

**Comment by maxamillion** [comment]
This is great info, thanks!

File: README.md (hunk 1 above)
Comment (sdodson) [inline README.md]: cleaned up stale branches.

**Pull request merged**